Corale.Colore/Core/NativeWrapper.cs

```diff
@@ -159,6 +159,11 @@ internal static Guid CreateMouseEffect(Razer.Mouse.Effects.Effect effect, IntPtr
         /// <remarks>This is currently only valid for <see cref="Razer.Mouse.Effects.Effect.None" />.</remarks>
         internal static Guid CreateMouseEffect(Razer.Mouse.Effects.Effect effect)
         {
+            if (effect == Razer.Mouse.Effects.Effect.None)
+            {
+                return CreateMouseEffect(effect, new Razer.Mouse.Effects.None(Razer.Mouse.Led.All));
+            }
```

Comment:
No braces needed here. 

It's a minor thing but our code tools format ifs with one statement to have no braces so it should be uniform across the codebase.